src/message-card/types.ts

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+import type { SyntheticEvent } from 'react';
+import type { Override } from '../helpers/overrides';
+import { KIND } from '../button';
+import { BACKGROUND_COLOR_TYPE, IMAGE_LAYOUT, OBJECT_FIT } from './constants';
+
+export type MessageCardOverrides = {
+  Root?: Override;
+  ContentContainer?: Override;
+  HeadingContainer?: Override;
+  ParagraphContainer?: Override;
+  Button?: Override;
+  Image?: Override;
+};
+
+export interface MessageCardProps {
+  onClick: (a: SyntheticEvent<HTMLButtonElement>) => unknown;
+  heading?: string;
+  paragraph?: string;
+  buttonLabel?: string;
```

Comment:
Design uses the word "Call to Action" instead of "Button". Semantically, "Call to Action" is more accurate since the entire MessageCard surface is a tap target, whereas the button itself is inert. But it seems to me that anyone familiar with Base Web would think of this element as "Button" since it does use the Base Web `Button` component. Open to suggestions.